iot-tenant-cleaner/override-dev.yaml

```diff
@@ -0,0 +1,4 @@
+osbs:
+      repository:
+            name: containers/amq-online-1
+            branch: amq7-amq-online-1-iot-tenant-cleaner-dev-rhel-7
```

Comment:
Remove this file, it is no longer used.

Reply:
I saw it in the other directories as well

Reply:
Hmm, yeah. We can either remove those, or modify this one to contain the correct information. The name and branch should be set to the name and branch of the comet build repo, which I don't think you've requested yet.